### ISSUE TITLE
Fine-tuning inception v3 - low learning rate

### ISSUE THREAD

**Issue by TiRune** [pytorch/vision#316]
Hey all,

I'm trying to fine-tune an inception v3 net in pytorch. I'm using this training code:
https://github.com/pytorch/examples/tree/master/imagenet
with this network:
https://github.com/pytorch/vision/blob/master/torchvision/models/inception.py

I didn't change much, just changed the input size to what inception expects, and I removed the aux-layers appendix.

Now, when retraining with either SGD or ADAM, I have to set the learning rate to 0.00001 for it to not completely kill itself. Which is weird, because usually learning rates are around 0.1 or 0.01 when initializing. When I train with even 0.0001 I get crashing performance until 0 precision. 

Anyone have an idea on why this happens? It seems weird to me that the learning rate has to be so incredibly low.

First few iterations for reference (with 0.0001 learning rate ):
Epoch: [0][0/40037]	Time 4.237 (4.237)	Data 0.553 (0.553)	Loss 0.8921 (0.8921)	Prec@1 81.250 (81.250)	Prec@5 96.875 (96.875)
Epoch: [0][10/40037]	Time 0.283 (0.644)	Data 0.000 (0.051)	Loss 0.7581 (1.1036)	Prec@1 87.500 (78.125)	Prec@5 96.875 (91.761)
Epoch: [0][20/40037]	Time 0.283 (0.472)	Data 0.000 (0.027)	Loss 0.6381 (1.1024)	Prec@1 81.250 (76.339)	Prec@5 93.750 (91.369)
Epoch: [0][30/40037]	Time 0.283 (0.412)	Data 0.000 (0.018)	Loss 1.2431 (1.1367)	Prec@1 71.875 (75.202)	Prec@5 84.375 (90.927)
Epoch: [0][40/40037]	Time 0.283 (0.381)	Data 0.000 (0.014)	Loss 1.1304 (1.1435)	Prec@1 75.000 (74.848)	Prec@5 93.750 (90.701)
Epoch: [0][50/40037]	Time 0.283 (0.361)	Data 0.000 (0.011)	Loss 0.6014 (1.1514)	Prec@1 84.375 (74.142)	Prec@5 93.750 (90.319)
Epoch: [0][60/40037]	Time 0.284 (0.349)	Data 0.000 (0.009)	Loss 1.3074 (1.1584)	Prec@1 75.000 (73.770)	Prec@5 81.250 (90.266)
Epoch: [0][70/40037]	Time 0.284 (0.339)	Data 0.000 (0.008)	Loss 1.9084 (1.1970)	Prec@1 53.125 (72.667)	Prec@5 75.000 (89.833)
Epoch: [0][80/40037]	Time 0.286 (0.333)	Data 0.000 (0.007)	Loss 1.2612 (1.2051)	Prec@1 71.875 (72.685)	Prec@5 87.500 (89.660)
Epoch: [0][90/40037]	Time 0.283 (0.327)	Data 0.000 (0.006)	Loss 1.1641 (1.2124)	Prec@1 75.000 (72.459)	Prec@5 96.875 (89.732)
Epoch: [0][100/40037]	Time 0.285 (0.323)	Data 0.000 (0.006)	Loss 0.9862 (1.2223)	Prec@1 71.875 (72.153)	Prec@5 93.750 (89.480)
Epoch: [0][110/40037]	Time 0.284 (0.320)	Data 0.000 (0.005)	Loss 0.8666 (1.2209)	Prec@1 78.125 (72.100)	Prec@5 96.875 (89.583)
Epoch: [0][120/40037]	Time 0.284 (0.317)	Data 0.000 (0.005)	Loss 1.1985 (1.2323)	Prec@1 75.000 (72.107)	Prec@5 87.500 (89.618)
Epoch: [0][130/40037]	Time 0.284 (0.314)	Data 0.000 (0.005)	Loss 1.3976 (1.2358)	Prec@1 71.875 (72.018)	Prec@5 84.375 (89.408)
Epoch: [0][140/40037]	Time 0.285 (0.312)	Data 0.000 (0.004)	Loss 1.2206 (1.2507)	Prec@1 78.125 (71.520)	Prec@5 93.750 (89.317)
Epoch: [0][150/40037]	Time 0.285 (0.310)	Data 0.000 (0.004)	Loss 0.9996 (1.2611)	Prec@1 75.000 (71.275)	Prec@5 90.625 (89.176)
Epoch: [0][160/40037]	Time 0.283 (0.309)	Data 0.000 (0.004)	Loss 1.2735 (1.2753)	Prec@1 68.750 (70.982)	Prec@5 93.750 (89.014)
Epoch: [0][170/40037]	Time 0.283 (0.307)	Data 0.000 (0.004)	Loss 1.6124 (1.2878)	Prec@1 65.625 (70.541)	Prec@5 87.500 (88.962)
Epoch: [0][180/40037]	Time 0.284 (0.306)	Data 0.000 (0.003)	Loss 2.1372 (1.3047)	Prec@1 59.375 (70.097)	Prec@5 78.125 (88.760)
Epoch: [0][190/40037]	Time 0.284 (0.305)	Data 0.000 (0.003)	Loss 1.4610 (1.3092)	Prec@1 65.625 (69.928)	Prec@5 87.500 (88.727)
Epoch: [0][200/40037]	Time 0.285 (0.304)	Data 0.001 (0.003)	Loss 1.3682 (1.3104)	Prec@1 68.750 (70.009)	Prec@5 93.750 (88.697)
Epoch: [0][210/40037]	Time 0.286 (0.303)	Data 0.000 (0.003)	Loss 1.3021 (1.3053)	Prec@1 71.875 (70.098)	Prec@5 90.625 (88.759)
And the precision just keeps on tanking from there without learning.

**Comment by alykhantejani** [comment]
Hi @TiRune,

This type of question is better for [the pytorch forums](https://discuss.pytorch.org/), I suggest you re-post it there - you're likely to get more engagement / suggestions there.
